docs/index.md

```diff
@@ -20,7 +20,7 @@ For more granular packages we recommend reading the documentation.
 
 # Features
 
-Features will be listed here.
+* [Telemetry (Serilog)](/features/telemetry-serilog) provides a set of additional functionality related to telemetry on the Serilog infrastructure.
```

Comment:
Same here, let's focus on enrichers only as most of this project will be about serilog